aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/widget/DialogWidgets.java

```diff
@@ -13,26 +13,9 @@
  */
 package com.exadel.aem.toolkit.core.handlers.widget;
 
-import java.lang.annotation.Annotation;
-import java.lang.reflect.Field;
-import java.util.Arrays;
-import java.util.Objects;
-import java.util.function.BiConsumer;
-
-import org.w3c.dom.Element;
-
+import com.exadel.aem.toolkit.api.annotations.container.AccordionPanel;
```

Comment:
(overall) please enable editorConfig plugin in your intellijIdea (if not already enabled) and refromat the imports again. 
currently, it seems like the formatting rules are not applied